.eslintignore

```diff
@@ -1,2 +1,3 @@
-dist
-node_modules
```

Comment:
Check to see if `node_modules` here was necessary

Reply:
It seems it is not. See https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file